tests/run-make/print-cfg/rmake.rs

```diff
@@ -0,0 +1,100 @@
+//! This checks the output of `--print=cfg`
+//!
+//! Specifically it checks that output is correctly formatted
+//! (ie. no duplicated cfgs, values are between "", names are not).
+//!
+//! It also checks that some targets have the correct set cfgs.
+
+extern crate run_make_support;
+
+use std::collections::HashSet;
+use std::ffi::OsString;
+use std::io::BufRead;
+use std::iter::FromIterator;
+
+use run_make_support::{rustc, tmp_dir};
+
+fn main() {
+    check(
+        /*target*/ "x86_64-pc-windows-gnu",
+        /*includes*/ &["windows", "target_arch=\"x86_64\""],
+        /*disallow*/ &["unix"],
+    );
```

Comment:
I suppose you could declare a struct to not need to have these comments, e.g.

```rs
struct Args {
    target: &'static str,
    includes: &'static [&'static str],
    disallow: &'static [&'static str],
}
```

then at call site it can become

```rs
check(Args {
    target: "x86_64-pc-windows-gnu",
    includes: &["windows", "target_arch=\"x86_64\""],
    disallow: &["unix"],
})
```

Reply:
I had the same idea but it felt overkill to me; but given you also seems to like it. I pushed a commit doing it a simple way, ie de-structuring right at the function arg. let me know what you think about it.